doc/en/parametrize.rst

```diff
@@ -81,17 +81,27 @@ them in turn:
 
 .. note::
 
-    pytest by default escapes any non-ascii characters used in unicode strings
-    for the parametrization because it has several downsides.
-    If however you would like to use unicode strings in parametrization and see them in the terminal as is (non-escaped), use this option in your ``pytest.ini``:
+    1. Parameter values are passed as-is to tests, and if you mutate them,
```

Comment:
Instead of using one note with a list inside, I think it will be better to use two notes.

Reply:
Should they be both in the same place, one after the other, or maybe move them into 2 different places to catch the attention of the reader?
Any suggestions?

Reply:
I think one after the other is fine, unless you think of a better place.

Reply:
Done.